### PR TITLE
Add Chinese translation for  CCGrid.h, CCGLBufferedNode.h and CCGrabber.h

### PR DESCRIPTION
**Pull request by zilongshanren**

### CODE REVIEW

**Comment by zilongshanren** [comment]
@pandamicro  please review this PR.
Thanks.

**Comment by pandamicro** [comment]
@andyque Your translations should be reviewed by @dabingnn

**Comment by zilongshanren** [comment]
@dabingnn  please review my PR, thanks

File: cocos/2d/CCGrid.h [hunk 1]
```diff
@@ -42,81 +42,111 @@ class Node;
...
 
-/** Base class for Other grid.
+/** @~english Base class for Other grid.
+ * @~chinese 其他网格类的基类。
 */
 class CC_DLL GridBase : public Ref
 {
 public:
-    /** create one Grid */
+    /** @~english create one Grid  @~chinese 创建一个网格*/
```
Comment (dabingnn) [inline cocos/2d/CCGrid.h]: It seems that params is missing here

File: cocos/2d/CCGrid.h [hunk 2]
```diff
@@ -131,29 +161,36 @@ class CC_DLL GridBase : public Ref
...
+/**@~english
+ * Grid3D is a 3D grid implementation. Each vertex has 3 dimensions: x,y,z
+ * @~chinese 
+ * Grid3D是一个3D网格实现。每个顶点都有三个维度:x,y,z
  */
 class CC_DLL Grid3D : public GridBase
 {
 public:
-    /** create one Grid. */
+    /** @~english create one Grid.  @~chinese 创建一个网格。*/
```
Comment (dabingnn) [inline cocos/2d/CCGrid.h:188]: param is missing here.

File: cocos/2d/CCGrid.h [hunk 3]
```diff
@@ -186,11 +227,19 @@ class CC_DLL Grid3D : public GridBase
     virtual void calculateVertexPoints() override;
     /**@}*/
     
-    /**@{
-     Getter and Setter for depth test state when blit.
+    /**@{@~english
+     * Set the depth test state when blit.
+     * @~chinese 
+     * 设置位块传送时的深度测试状态。
```
Comment (dabingnn) [inline cocos/2d/CCGrid.h]: The translation of `blit` seems a little strange. Could we use the original `blit` instead?

File: cocos/2d/CCGrid.h [hunk 4]
```diff
@@ -203,55 +252,71 @@ class CC_DLL Grid3D : public GridBase
...
+    /** @~english Returns the tile at the given position.
+     * @~chinese 返回给定位置的瓦片。
      * @js NA
      * @lua NA
      */
     Quad3 getTile(const Vec2& pos) const;
-    /** returns the tile at the given position 
+    /** @~english returns the tile at the given position 
+     * @~chinese 返回给定位置的瓦片.
+     * @deprecated Use getTile instead.
```
Comment (dabingnn) [inline cocos/2d/CCGrid.h:309]: @pandamicro If I remember correctly, deprecated function do not need a doc. Please confirm it. Thanks.

Comment (pandamicro) [inline cocos/2d/CCGrid.h:309]: @deprecated tag don't need to be translated, but deprecated functions should be documented too

Comment (dabingnn) [inline cocos/2d/CCGrid.h:309]: OK, this standard is different from the previous, we need to write it down.

Comment (pandamicro) [inline cocos/2d/CCGrid.h:309]: I will clarify it in the doc

File: cocos/2d/CCGrid.h (hunk 3 above)
Comment (zilongshanren) [inline cocos/2d/CCGrid.h]: ok, let's use blit instead.

**Pull request merged**